index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="/public/LOGO.svg" />
```

Comment:
```suggestion
    <link rel="icon" type="image/svg+xml" href="/LOGO.svg" />
```

Le dossier `public` se trouve à la racine du lien `/`